packages/link-modules/src/definitions/product-variant-price-set.ts

```diff
@@ -25,6 +25,7 @@ export const ProductVariantPriceSet: ModuleJoinerConfig = {
       isInternalService: true,
       primaryKey: "id",
       foreignKey: "variant_id",
+      // TODO: Remove this when product module is the default product service
```

Comment:
question: The alias? Is this not related to `isInternalService`?

Reply:
yeah, this seems to just be a bad merge